packages/workbox-build/src/entry-points/generate-sw.js

```diff
@@ -17,13 +17,16 @@
 const path = require('path');
 
 const cdnUtils = require('../lib/cdn-utils');
-const copyWorkboxSW = require('../lib/copy-workbox-sw');
+const copyWorkboxLibraries = require('../lib/copy-workbox-libraries');
```

Comment:
Can you export this method as part of the `workbox-build` API such that anyone using the injectManifest can easily get all the files.

Reply:
Yup. I'll add some JSDocs as well.